build_projects/dotnet-cli-build/SetEnvVar.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class SetEnvVar : Task
```

Comment:
:(
You are correct that there isn't an MSBuild task that sets environment variables but I would argue that we shouldn't need one as depending on environment variables is always a bad idea. I think we can live with this temporarily but we should really try and avoid using environment variables.